tests/ui/unnecessary_sort_by.rs

```diff
@@ -9,7 +9,6 @@ fn unnecessary_sort_by() {
 
     let mut vec: Vec<isize> = vec![3, 6, 1, 2, 5];
     // Forward examples
-    vec.sort_by(|a, b| a.cmp(b));
```

Comment:
We should keep this test until we implement unstable sort lint for `sort_by` (SortingKind::ByCmp)